catroid/src/main/java/org/catrobat/catroid/content/actions/GoToRandomPositionAction.kt

```diff
@@ -20,40 +20,23 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-
-package org.catrobat.catroid.content.actions;
-
-import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction;
-
-import org.catrobat.catroid.common.ScreenValues;
-import org.catrobat.catroid.content.Sprite;
-
-public class GoToRandomPositionAction extends TemporalAction {
-
-	private Sprite sprite;
-	private float randomXPosition;
-	private float randomYPosition;
-
-	@Override
-	protected void update(float percent) {
-		randomXPosition = (float) Math.random()
-				* (ScreenValues.SCREEN_WIDTH + 1) - (ScreenValues.SCREEN_WIDTH / 2);
-		randomYPosition = (float) Math.random()
-				* (ScreenValues.SCREEN_HEIGHT + 1) - (ScreenValues.SCREEN_HEIGHT / 2);
-
-		sprite.look.setXInUserInterfaceDimensionUnit(randomXPosition);
-		sprite.look.setYInUserInterfaceDimensionUnit(randomYPosition);
-	}
-
-	public void setSprite(Sprite sprite) {
-		this.sprite = sprite;
-	}
-
-	public float getRandomXPosition() {
-		return this.randomXPosition;
-	}
-
-	public float getRandomYPosition() {
-		return this.randomYPosition;
-	}
-}
+package org.catrobat.catroid.content.actions
+
+import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction
+import org.catrobat.catroid.common.ScreenValues
+import org.catrobat.catroid.content.Sprite
+
+class GoToRandomPositionAction : TemporalAction() {
+    var sprite: Sprite? = null
+    var randomXPosition: Float = Math.random().toFloat() * (ScreenValues.SCREEN_WIDTH + 1) -
+        (ScreenValues.SCREEN_WIDTH / 2)
```

Comment:
Please consider my suggestion to import the members SCREEN_WIDTH and SCREEN_HEIGHT directly using ALT + ENTER and then import members from ...